src/Jackett.Common/Definitions/bibliotik.yml

```diff
@@ -0,0 +1,146 @@
+---
+  site: bibliotik
+  name: Bibliotik
+  description: "Bibliotik is a private site for eBooks and audiobooks"
+  language: en-US
+  type: private
+  encoding: UTF-8
+  links:
+    - https://bibliotik.me/
+
+  caps:
+    categorymappings:
+      - { id: 1, cat: PC, desc: "Applications" }
+      - { id: 3, cat: Audio/Audiobook, desc: "Audiobooks" }
+      - { id: 4, cat: Books/Comics, desc: "Comics" }
+      - { id: 5, cat: Books/Ebook, desc: "eBooks" }
+      - { id: 7, cat: Books/Magazines, desc: "Magazines" }
+
+    modes:
+      search: [search]
+
+  settings:
+    - name: username
+      type: text
+      label: Username
+    - name: password
+      type: password
+      label: Password
+    - name: orderby
+      type: select
+      label: Sort requested from site
+      default: "relevance"
+      options:
+        "relevance": "relevance"
+        "year": "year"
+        "size": "size"
+        "duration": "duration"
+        "added": "added"
+        "leechers": "leechers"
+        "seeders": "seeders"
+        "snatches": "snatches"
+        "comments": "comments"
+        "files": "files"
+    - name: order
+      type: select
+      label: Order requested from site
+      default: "desc"
+      options:
+        "desc": "desc"
+        "asc": "asc"
+
+  login:
+    path: /
+    method: form
+    form: form#loginform
+    submitpath: /
+    cookies: ["__cfduid", "id"]
```

Comment:
should the cookies be below in the inputs section?

Reply:
I was under the impression that this just indicated which cookies should be used to associate a session with later queries. If that's not what it's intended for, then no, I don't believe we need them as inputs.